src/model_io/urdf-kdl/external/urdfdom_headers/urdf_model/include/urdf_model/boost_replacements.h

```diff
@@ -48,24 +48,29 @@ namespace urdf {
  * Convert a string to a double
  *
  */
-bool inline stringToDouble(const std::string & inStr, double & outDouble)
+bool inline stringToDoubleWithClassicLocale(const std::string & inStr, double & outDouble)
 {
-    outDouble = std::atof(inStr.c_str());
-    return true;
+    std::istringstream ss(inStr);
+    ss.imbue(std::locale::classic());
+    ss >> outDouble;
+    return !(ss.fail());
 }
 
-bool inline stringToInt(const std::string & inStr, int & outInt)
+bool inline stringToIntWithClassicLocale(const std::string & inStr, int & outInt)
```

Comment:
Just a curiosity, why not using a templated function now that inside there are no differences between the `toDouble` and `toInt` and `toUnsigned` functions?